src/Chapter1.hs

```diff
@@ -542,7 +546,8 @@ value after "=" where the condition is true.
 Casual reminder about adding top-level type signatures for all functions :)
 -}
 
-mid x y z = error "mid: not implemented!"
+mid :: Int -> Int -> Int -> Int
+mid x y z = (x + y + z) - (min x (min y z)) - (max x (max y z))
```

Comment:
That's a very clever implementation! :brain: 
Though, I wonder, will it still work around big numbers because of integer overflow :thinking: 
We don't test such cases in our course, but something to keep in mind when implementing it in real life :slightly_smiling_face:

Reply:
You're right my solution will break if all 3 numbers add up to more than MAX_INT (or haskell equivalent), I'll change it to a few guards.